### PR TITLE
update waxholm to newest version

### PR DESCRIPTION
**Pull request by PolarBean**
Here we upgrade the waxholm rat atlas to be the newest version. Addressing https://github.com/brainglobe/brainglobe-atlasapi/issues/215

### CODE REVIEW

**Review by alessandrofelder — CHANGES_REQUESTED** [review]
Thanks a lot for this @PolarBean !

I have two minor suggestions that I think we need

For completeness, I regenerated the atlas locally and it passes the validation functions defined in `validate_atlases.py` apart from `validate_mesh_matches_image_extents`, due to spurious pixels, which is a known issue https://github.com/brainglobe/brainglobe-atlasapi/issues/211 and we can tackle separately.

File: brainglobe_atlasapi/atlas_generation/atlas_scripts/whs_sd_rat.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-__version__ = "0"
-
+__version__ = "1"
```
[the remark this answers is not in the document]
Comment (PolarBean) [reply brainglobe_atlasapi/atlas_generation/atlas_scripts/whs_sd_rat.py:1]: no worries!

**Comment by PolarBean** [comment]
out of curiousity, is the spurious pixel issue an issue with the atlas itself? If it is I'll let the atlas developers know as they will likely want to fix that.

**Comment by alessandrofelder** [comment]
> an issue with the atlas itself?

Yes, as far as I can tell the annotation image has some non-zero pixels outside the brain:

**Review by alessandrofelder — APPROVED** [review]

**Comment by aeidi89** [comment]
> > an issue with the atlas itself?
> 
> Yes, as far as I can tell the annotation image has some non-zero pixels outside the brain:
> 
> image

Thank you @alessandrofelder for visualising this, and thanks @PolarBean for asking! I'll put it on our to do list for the atlas.

**Comment by alessandrofelder** [comment]
@aeidi89 and @PolarBean would you like me to upload this version of the atlas to GIN now, or wait until you fixed the spurious pixel issue? (Happy either way)

**Comment by PolarBean** [comment]
Im happy for it to go now, i think the spurious pixel issue will be fixed at a later date.

**Comment by adamltyson** [comment]
For ref, this is the issue https://github.com/brainglobe/brainglobe-atlasapi/issues/211

**Pull request merged**